### PR TITLE
Get Correct Number of Admins for Child Collective When Showing the Freeze Banner

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/6598

[no-freeze-banner.webm](https://user-images.githubusercontent.com/12435965/227112221-f40c9663-4230-4546-ac4a-b2fc349cf816.webm)

### CODE REVIEW

**Comment by Betree** [comment]
Passing the review to @kewitz

File: components/collective-page/CollectiveNotificationBar.js [hunk 1]
```diff
@@ -98,6 +98,9 @@ const messages = defineMessages({
 });
 
 const getNotification = (intl, status, collective, host, LoggedInUser, refetch) => {
+  const numberOfAdmins = [CollectiveType.EVENT, CollectiveType.PROJECT].includes(collective.type)
+    ? collective?.parentCollective?.coreContributors?.filter(c => c.isAdmin)?.length
```
Comment (kewitz) [inline components/collective-page/CollectiveNotificationBar.js:102]: These also include `MEMBER`.
We can either add the `admin` property to this query or update the graphqlV1 members resolver to return inherited admins.

Comment (kewitz) [inline components/collective-page/CollectiveNotificationBar.js:102]: Now that children collectives can have their own admins, we still need to sum this with `collective.admins.lenght`.

File: components/collective-page/CollectiveNotificationBar.js [hunk 2]
```diff
@@ -98,6 +98,9 @@ const messages = defineMessages({
 });
 
 const getNotification = (intl, status, collective, host, LoggedInUser, refetch) => {
+  const numberOfAdmins = [CollectiveType.EVENT, CollectiveType.PROJECT].includes(collective.type)
```
Comment (kewitz) [inline components/collective-page/CollectiveNotificationBar.js:101]: I think checking for `parentCollective` should be enough.
```suggestion
  const numberOfAdmins = collective.parentCollective
```

**Review by kewitz — APPROVED** [review]
Nice work. Left some suggestions here but feel free to merge after considering `collective.admins` as well.

File: components/collective-page/CollectiveNotificationBar.js (hunk 1 above)
Comment (SudharakaP) [reply components/collective-page/CollectiveNotificationBar.js]: Thanks. Corrected. 👍🏽

**Pull request merged**